Concepts/InMemoryKeyValueStore/Entities/KeyValueStore.cs

```diff
@@ -0,0 +1,84 @@
+﻿using InMemoryKeyValueStore.Services;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+
+namespace InMemoryKeyValueStore.Entities
+{
+    public class KeyValueStore
+    {
+
+        private Dictionary<string, ValueStore> _keyValue;
```

Comment:
class ValueStore can be an inner class since the valueStore cannot exist without the existence of the keyvalue store. Same thing goes for value as well.

Reply:
Done. Please take a look